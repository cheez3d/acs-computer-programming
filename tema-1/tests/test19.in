lalea zambila trandafir garoafa
45552344466655520222#26335554443304887772055533885558844408666766677727777
floarea mea preferata este zambila
3
mea ta
preferata iubita
zambila laleaua
mac ghiocel flori
