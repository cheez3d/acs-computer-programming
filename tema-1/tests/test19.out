55525553320999926224445552087772663233344477704277766623332
gladiola camelie gura leului toporas
33355566627773320820444882244482033777783305552555332882
3
62222 44444466622233555 333555666777444 
44444466622233555 2
0
0
